docs/application-connector/02-03-proxy-service.md

```diff
@@ -3,6 +3,8 @@ title: Application Gateway
 type: Architecture
 ---
 
+> **CAUTION:** This document describes the Application Gateway working in the default legacy [mode](#architecture-application-connector-components-application-operator). To read about the Application Gateway working in the alternative **gatewayOncePerNamespace** mode, see [this](#details-application-gateway) document. 
```

Comment:
```suggestion
> **CAUTION:** This document describes the Application Gateway working in the [default legacy mode](#architecture-application-connector-components-application-operator). To read about the Application Gateway working in the alternative **gatewayOncePerNamespace** mode, see [this](#details-application-gateway) document. 
```

Reply:
I scoped it to `mode`, as I mention two modes in the note and didn't want to link to their description twice.